README.md

```diff
@@ -249,7 +231,7 @@ We welcome code contributions through pull requests from the community.
 * Please use [gofmt](https://golang.org/cmd/gofmt/) for formatting the code before submitting the pull request.
 
 # Using JFrog CLI
-JFrog CLI can be used for a variety of functions with Artifactory, Bintray, Xray and Mission Control,
+JFrog CLI can be used for a variety of functions with Artifactory, Xray and Mission Control,
```

Comment:
JFrog CLI can be used for a variety of functions with Artifactory, Xray and Mission Control,
-->
JFrog CLI can be used for a variety of functions with JFrog Artifactory, Xray, Distribution and Mission Control,